src/openzaak/components/zaken/openapi.yaml

```diff
@@ -3576,8 +3575,8 @@ components:
           title: Inp locatiebeschrijving
           type: string
           maxLength: 1000
+      nullable: true
```

Comment:
this is diverting from the standard spec -> investigate why this is happening

Reply:
these are missing in the standard, but the reference implementation _does_ allow null

Reply:
Updated reference implementation here as well: https://github.com/VNG-Realisatie/zaken-api/pull/196